algosdk/atomic_transaction_composer.py

```diff
@@ -449,13 +449,13 @@ def gather_signatures(self) -> list:
             raise error.AtomicTransactionComposerError(
                 "missing signatures, got {}".format(stxn_list)
             )
-        full_stxn_list = cast(List[SignedTransaction], stxn_list)
+        full_stxn_list = cast(Sequence[SignedTransaction], stxn_list)
 
         self.status = AtomicTransactionComposerStatus.SIGNED
-        self.signed_txns = full_stxn_list
+        self.signed_txns = list(full_stxn_list)
```

Comment:
@algochoi In https://github.com/jdtzmn/py-algorand-sdk/pull/3/commits/8644dffd073ba7b5925be0d8918aea588022a77a, I reverted the change.  Resolving because the thread no longer applies.